CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -2,11 +2,32 @@ import { useState, useEffect } from "react"
 
 export default function Report({ reportID }) {
   const [results, setResults] = useState(null)
+  const [statistics, setStatistics] = useState(null)
+  const [showDiseaseStats, setShowDiseaseStats] = useState(false)
 
   useEffect(() => {
     // Here you will fetch the report data from the API based on the reportID
+    const fetchReportStatistics = async () => {
+		try {
+			const response = await fetch(`http://localhost:8000/report_statistics/${reportID}`)
```

Comment:
may not want to hardcode the endpoint, could use a global variable to identify the beginning of the endpoint as its gonna change when everything goes live